packages/ChangesReporting/ValueObject/RectorWithFileAndLineChange.php

```diff
@@ -41,6 +42,33 @@ public function getRectorClass(): string
         return get_class($this->rector);
     }
 
+    public function getRectorClassWithChangelogUrl(): string
+    {
+        $rectorClass = get_class($this->rector);
+
+        /* @phpstan-ignore-next-line */
```

Comment:
We put ignores to `phsptan.neon`, to keep track on them and avoid code polution with tool ignores. Could you move it there?

Reply:
Fixed by: https://github.com/rectorphp/rector/pull/6078